source/release_notes/5.0.2-1.rst

```diff
@@ -0,0 +1,27 @@
+.. _PSMDB-5.0.2-1:
+
+================================================================================
+*Percona Server for MongoDB* 5.0.2-1 (Release Candidate)
+================================================================================
+
+:Date: August 16, 2021
+:Installation: `Installing Percona Server for MongoDB <https://www.percona.com/doc/percona-server-for-mongodb/5.0/install/index.html>`_
+
+We are pleased to announce the release candidate of Percona Server for MongoDB 5.0.2-1. 
+
+Percona Server for MongoDB 5.0.2-1 is an enhanced, open source, and highly-scalable database that is a
+fully-compatible, drop-in replacement for MongoDB 5.0.2 Community Edition. It includes `all features of MongoDB 5.0.2 Community Edition <https://docs.mongodb.com/v5.0/release-notes/5.0/#5.0.2---aug-06--2021>`_ and provides :ref:`enterprise-level enhancements <compare>` for free.
+
+The following are most notable features introduced in MongoDB 5.0.2 Community Edition:
```

Comment:
```suggestion
The following are most notable features introduced in Percona Server for MongoDB 5.0:
```

Reply:
But these are features in MongoDB CE not in Percona Server for MongoDB, current wording seems to be more accurate and it was previously used:
https://www.percona.com/doc/percona-server-for-mongodb/4.0/release_notes/4.0.4-1.html
https://www.percona.com/doc/percona-server-for-mongodb/4.2/release_notes/4.2.0-1.html

Reply:
Well, by these being features of MongoDB CE, they are features of PSMDB as well. I've updated the proposed change, hopefully that makes it more clear.